### DISCUSSION TITLE
display table with two conditions using findby

### DISCUSSION THREAD

**Discussion by SafaeZer** [symfony/symfony#45552] [Q&A]
I displayed the tickets table by urgency and now I need to display only the not assigned tickets (responsible=null)
I've tried to add another array which is ->findBy([], ['priority' => 'DESC'], ['responsible' => null]) but it doesn't work
this is my controller 
` public function index(TicketRepository $ticketRepository): Response
    {
        $this->denyAccessUnlessGranted('ROLE_SUPPORT');

        return $this->render('support/supportTable.html.twig', [
            'tickets' => $ticketRepository->findBy([], ['priority' => 'DESC']),
        ]);
    }`

**Comment by craigh** [comment]
google is your friend

https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/working-with-objects.html#by-simple-conditions

**Comment by chyshyya** [comment]
$ticketRepository->findBy(['responsible' => null], ['priority' => 'DESC'])
